packages/atlas-react-fetch-loader/src/FetchLoader.js

```diff
@@ -66,8 +66,12 @@ const withFetchLoader = (WrappedComponent) => {
           throw new Error(`${url} => ${response.status}`)
         }
 
+        const data = await response.json()
+        Object.keys(this.props.renameDataKeys)
+          .forEach(key => delete Object.assign(data, {[this.props.renameDataKeys[key]]: data[key] })[key])
```

Comment:
This is a one liner that does the following:
```javascript
// Add a new key which points at the old key contents: data[newKey] === data[oldKey]
Object.assign(data, { [newKey]: data[oldKey] })
// Remove old key
delete data[oldKey]
```

Reply:
Cool! A clean and simple!